lib/castle/client.rb

```diff
@@ -4,19 +4,28 @@ module Castle
   class Client
     class << self
       def from_request(request, options = {})
-        new(to_context(request, options), options[:do_not_track])
+        new(
+          to_context(request, options),
+          to_options(options)
+        )
       end
 
-      def to_context(request, options = {})
-        default_context = Castle::DefaultContext.new(request, options[:cookies]).call
-        Castle::ContextMerger.new(default_context).call(options[:context] || {})
+      def to_context(request, context_options = {})
```

Comment:
why renaming to `context_options`, I think if any I would call it `additional_context` see other methods